buildSrc/src/main/groovy/com/uber/okbuck/generator/BuckFileGenerator.groovy

```diff
@@ -97,7 +94,7 @@ final class BuckFileGenerator {
                     }
                     break
                 default:
-                    break
+                    throw new IllegalArgumentException("ProjectType not handled: $projectType")
```

Comment:
I strongly think that throwing an exception is the best option here. Otherwise when a developer adds a new ProjectType to the enum, it will silently fail. This took me a while to debug when I added a new `KotlinAndroidModule` as a project type.